scripts/REST_Axion_PlotResonances.C

```diff
@@ -0,0 +1,123 @@
+///////////////////////////////////////////////////////
+/// \file      AxionPlotResonances.C
+/// \brief     This script plots the transmission probability as a function of the axion mass for a given gas
+/// and axion energy until a maximum axion mass.
+// Arguments by default are (in order):
+// -ma_max = 0.1 eV (axion mass)
+// -Ea = 4.2 keV (axion energy)
+//  gasName = "He" (Gas name)
+//  vacuum = true (Vacuum added to the sum or not)
+//  n_ma = 100000 (Number of points to be plotted)
+//
+// Example of usage:
+// restRoot -l
+// .L AxionPlotResonances.C
+// AxionPlotResonances(0.1, 4.2, "He", true, 100000)
+//
+///////////////////////////////////////////////////
+/// \date       11/12/2023
+/// \author     Francisco R. Candon
+///////////////////////////////////////////////////
+
+int REST_Axion_PlotResonances(double ma_max = 0.1, double Ea = 4.2, std::string gasName = "He",

```

Comment:
Perhaps it is possible to reduce the arguments. `Bool_t vacuum = true` may not be necessary. We could accept the argument `gasName="vacuum"`. Then, you could do:

```
TRestAxionField* ax = new TRestAxionField();
TRestAxionBufferGas* gas = new TRestAxionBufferGas();

if( gas == "vacuum")
{
    ax->AssignBufferGas(nullptr);
}
else
{
        gas->SetGasDensity(gasName, density[i]);
        ax->AssignBufferGas(gas);
}
````

Reply:
I'm not using it that way, if  `Bool_t vacuum = true ` the vector that contains the sum of all the probabilities also adds the vacuum, if it is false `Bool_t vacuum = falso` is not added.

Reply:
I see, then, perhaps you could rename the argument name to `addVacuumResonanceToPlot`.